daffodil-runtime1-layers/src/test/scala/org/apache/daffodil/layers/runtime1/TestLengthLimitedLineFoldingStreams.scala

```diff
@@ -59,19 +60,9 @@ esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat\r
     //
     // regex is CRLF not followed by a tab or space.
     //
-    val rls = new RegexLimitingStream(bba, "\\r\\n(?!(?:\\t|\\ ))", "\r\n", iso8859)
+    val rls = new RegexLimitingInputStream(bba, "\\r\\n(?!(?:\\t|\\ ))", "\r\n", iso8859)
```

Comment:
Similar question as above, should these pattern strings be moved to a common constant?